src/item.cpp

```diff
@@ -545,6 +545,11 @@ bool item::is_unarmed_weapon() const
     return has_flag( "UNARMED_WEAPON" ) || is_null();
 }
 
+bool item::is_frozen_liquid() const
+{
+    return ( has_flag( "FROZEN" ) && made_of_from_type( LIQUID ) ) || is_null();
```

Comment:
```suggestion
    return has_flag( "FROZEN" ) && made_of_from_type( LIQUID );
```
Is there any specific reason why this needs to check for null items explicitly?